src/components/okta_sign_in_widget_wrapper.js

```diff
@@ -0,0 +1,19 @@
+// Wrapper for the Okta Sign In Widget
+
+import React, { Component } from 'react';
+import OktaSignInWidget from './okta_sign_in_widget';
+
+// Define here your parameters
+const WIDGET_CONTAINER_ID = 'okta-login-container';
+const LOGIN_SUCCESS_ROUTE = '/home';
+const OPTIONS = {
+            baseUrl: "https://dev-570863.oktapreview.com",
+            clientId : "OaCz3jBQxbaEnsDAFO3A",
+            redirectUri: "http://localhost:8080/"
+          };
+
+export default class OktaSignInWidgetWrapper extends Component {
+  render() {
+    return <OktaSignInWidget container={WIDGET_CONTAINER_ID} options={OPTIONS} successRoute={LOGIN_SUCCESS_ROUTE} />;
```

Comment:
Does everyone use the react router? I'm wondering if a better interface would be just to have them handle the success function themselves? I.e. they pass in a callback function here that gets invoked with the res, and they choose what to do with it.